docs/pact-reference.md

```diff
@@ -1184,7 +1184,7 @@ Define NAME as VALUE, with option DOC-OR-META. Value is evaluated upon module lo
 (defpact NAME ARGLIST [DOC-OR-META] STEPS...)
 ```
 
-Define NAME as a _pact_, a multistep computation intended for private transactions.
+Define NAME as a _pact_, a multistep transaction computation.
```

Comment:
Good catch. Suggest "a computation comprised of steps that happen in distinct transactions."